django/core/management/commands/shell.py

```diff
@@ -111,7 +126,22 @@ def python(self, options):
         # Start the interactive interpreter.
         code.interact(local=imported_objects)
 
+    def get_namespace(self):
+        apps_models = apps.get_models()
+        imported_objects = {}
```

Comment:
Call this variable `namespace`, throughout
```suggestion
        namespace = {}
```